web/html/src/utils/network.ts

```diff
@@ -64,7 +64,7 @@ function request<Returns>(
 function post<Returns = any, Payload = any>(
   url: string,
   data?: DataType<Payload>,
-  contentType: string = "application/json",
+  contentType: string | boolean = "application/json",
```

Comment:
Perhaps this change is left over from your previous changes? Right now it doesn't make sense since you could end up with content types such as `"false; charset=UTF-8"`